README.md

```diff
@@ -6,7 +6,7 @@ Use command as a placeholder where WSL path for the `workspaceFolder` is require
 
 ## Features
 
-This extension returns the `workspaceFolder` path, converted from Windows path format to WSL path format. For example, a `workspaceFolder` with a windows path of `c:\\Users\\Me\\Projects\\project` would return a WSL path of `/mnt/c/Users/Me/Projects/project`.
+This extension returns some VS Code workspace file variables, converted from Windows path format to WSL path format. For example, a `workspaceFolder` with a windows path of `c:\\Users\\Me\\Projects\\project` would return a WSL path of `/mnt/c/Users/Me/Projects/project`.
```

Comment:
I'm not sure I fully understand your use case. Please explain the purpose of this new feature in more detail.

Under `Usage` add more details about how you expect this new feature to be used, with code/settings examples. Also, is there any known issues?

Reply:
✔